modules/serverless-logic-upgrade-redeploying-workflows-with-dev-profile.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.34-to-1.35
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-redeploying-workflows-with-dev-profile_{context}"]
+= Re-deploying workflows with the `dev` profile
```

Comment:
I know this already went through peer review, but [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-prefixes) suggests that redeploy shouldn't be hyphenated. I'll just mark once but it appears in other modules too.

see previous comment regarding backticks in module headings

Reply:
Yes, done!